src/Symfony/Component/Serializer/Normalizer/DateIntervalNormalizer.php

```diff
@@ -118,16 +108,17 @@ public function denormalize($data, string $type, string $format = null, array $c
         }
     }
 
-    /**
-     * {@inheritdoc}
```

Comment:
let's revert all those changes, they're false positive from the cs fixer on branch 5.4

Reply:
fixed